sous-chef/sous_chef/recipe_book/_recipe_book.py

```diff
@@ -186,3 +189,31 @@ def create_timedelta(row_entry):
             )
 
         return time_converted
+
+
+def is_cache_valid(metadata) -> bool:
+    time_cache = metadata["time"]
+    time_recipe_files = max(
+        recipe.stat().st_mtime
+        for recipe in RECIPE_BOOK_PATH.glob(RECIPE_FILE_PATTERN)
+    )
+    if time_recipe_files > time_cache:
```

Comment:
the downside to this comparison, is if we have the case where the cache is newer. so for instance, I had been testing a cache from July 8 files, so when I synced with dropbox for the July 18 files, the cache was newer. I think though this is an ok edge case. You?

Reply:
Yes, I think this is an edge case we can live with. This can only happen if dropbox syncing is not always active, right?